CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support for `AtomicCmpExchange128` intrinsic.
+- Support for `GetTypeKind` intrinsic.
+- Support for `OpenString` intrinsic type.
+
+### Changed
+
+- Improve `VarArg*` intrinsic signatures and make them stricter.
```

Comment:
This is pretty vague and also probably fits into Fixed rather than Changed. 
Maybe

```suggestion
- Improve type modelling around the `VarArg*` intrinsic routines.
```

Reply:
Sure.
It should probably still be in `Changes` - it's a pure QA/accuracy change, since there was no way for the previous modeling to result in invocation resolution failures.